assets/cubyz/recipes/other_recipes

```diff
@@ -0,0 +1,4 @@
+cubyz:stone
+cubyz:chisel
+result cubyz:stone_bricks
+result cubyz:chisel
```

Comment:
Maybe we could add these after #760? because I think this behavior might cause problems.
Alternatively, maybe just add the brick recipes without the chisel for now.